core/msd/src/main/rdl/TransportPolicyRule.tdl

```diff
@@ -73,4 +87,15 @@ type TransportPolicyEgressRule Struct {
 type TransportPolicyRules Struct {
     Array<TransportPolicyIngressRule> ingress; // List of ingress rules
     Array<TransportPolicyEgressRule> egress; // List of egress rules
-}
\ No newline at end of file
+}
+
+type TransportPolicyValidateRule Struct {
+    Array<TransportPolicyPeer> source;
+    Array<TransportPolicyPeer> destination;
```

Comment:
Destination does not need to be an array right? its always going to one service identity for a given policy? Would TransportPolicyEntitySelector make a better sense here?